cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -118,6 +118,10 @@ func RemoveStackedEtcdMemberFromCluster(client clientset.Interface, cfg *kubeadm
 	klog.V(2).Infof("[etcd] get the member id from peer: %s", etcdPeerAddress)
 	id, err := etcdClient.GetMemberID(etcdPeerAddress)
 	if err != nil {
+		if errors.Is(etcdutil.ErrNoMemberIDForPeerURL, err) {
+			klog.V(5).Infof("[etcd] member was already removed, because no member id exists for peer %s", etcdPeerAddress)
+			return nil
+		}
```

Comment:
The `RemoveMember` will handle `ErrNoMemberIDForPeerURL` later in the function.

Can we pass through so that the log of removing an etcd member will be printed and for file delete, not-found is a successful state? I don't have a strong opinion on this, but I believe it would make the log more readable for users.

Reply:
This change is inside `func RemoveStackedEtcdMemberFromCluster`.

With the current change, the log (if verbose is less than 5) will show only `[etcd] get the member id from peer: %s` and return.
I prefer to change the log level of line 122 higher so that users can know why.

Sorry. My last comment was wrong. If there is an error here, the remove member will use an empty ID which is odd.

Reply:
Thanks for rephrasing!

> I prefer to change the log level of line 122 higher so that users can know why.

I chose log level 5 for two reasons. First, because it's not an error, so I don't want to alert the user unless they've asked for increased verbosity. Second, because all log messages that indicate a problem (i.e. start with `Failed`) use level 5.

Of course I can reduce the log level, if you feel strongly. Let me know.

Reply:
/lgtm
overall